README.md

```diff
@@ -350,9 +350,15 @@ TEST_ACMPCA_EXPENSIVE_ENABLE=1 go test -v ./...
 Every source file in this project should be formatted with `go fmt`.
 
 ### Releasing new versions
+We try to follow the release process as deifned in our [Coding Methodology](https://www.notion.so/gruntwork/Gruntwork-Coding-Methodology-02fdcd6e4b004e818553684760bf691e#08b68ee0e19143e89523dcf483d2bf48). _This document is only visible to Gruntwork employees currently._
```

Comment:
The coding methodology document is actually externally visible, as it says on the top of that Notion page.

Reply:
Oh good point! I saw the settings on the page, but interpreted them wrong!